guides/source/association_basics.md

```diff
@@ -2724,3 +2724,156 @@ will run a query like:
 ```sql
 SELECT "vehicles".* FROM "vehicles" WHERE "vehicles"."type" IN ('Car')
 ```
+
+Delegated types
```

Comment:
Headings should be mostly [capitalized](https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings).
```suggestion
Delegated Types
```

Reply:
Fixed

https://github.com/rails/rails/pull/46190/files#diff-d977574b785ce95fe656c58211c668992bf95214d52d3ffadb409ef8f69ad287R2728